src/Main.purs

```diff
@@ -18,8 +19,14 @@ main = launchAff_ $ runAppM app =<< liftEffect (execParser options)
   where
   app = do
     { config, cssInput, cssOutput, output } <- ask
-    info empty "Building Tailwind css"
+    info
+      ( tag "Tailwind config file" config
+          <> tag "Css input file" cssInput
+          <> tag "Css output file" cssOutput
+      )
+      "Generating Tailwind css..."
```

Comment:
So this might result in something like:
```
[INFO] 2020-10-13T15:05:04.011Z Generating Tailwind css...
   Css input file: /Users/dandoh/scoville/tailwind-generator/assets/input.css
   Css output file: ./tailwind.css
   Tailwind config file: 
[INFO] 2020-10-13T15:05:05.951Z Done generating Tailwind css.
[INFO] 2020-10-13T15:05:06.959Z Done   
```
(with colors of course)

Reply:
Looks really good to me 😄